symengine/eval_mpc.cpp

```diff
@@ -232,15 +232,23 @@ class EvalMPCVisitor : public BaseVisitor<EvalMPCVisitor> {
         } else if (x.__eq__(*E)) {
             mpfr_t t;
             mpfr_init2(t, mpc_get_prec(result_));
-            mpfr_const_euler(t, rnd_);
+            mpfr_t one_;
+            mpfr_init(one_);
```

Comment:
You are initializing `one_` to default precision which is not the same as the precision of `result_`